packages/component-submission/client/SubmissionOperations.js

```diff
@@ -2,15 +2,15 @@ import { cloneDeep } from 'lodash'
 import { graphql, compose } from 'react-apollo'
 import React from 'react'
 import omitDeep from 'omit-deep-lodash'
-import Loading from 'ui/atoms/Loading'
-// TODO: This should not be importing from the other component
 import { ALL_MANUSCRIPTS } from '@elifesciences/component-dashboard/client/graphql/queries'
+import Loading from '../../../app/components/ui/atoms/Loading'
```

Comment:
does this need to change? would the ui alias not still work here?

Reply:
It doesn't need to change, the alias should work just fine.